README.md

```diff
@@ -67,19 +69,24 @@ TIA & PLCs:
 - PLC 1512SP-1 PN FW V2.1
 
 Industrial Edge:
-- Industrial Edge Management V1.7.4
-- Industrial Edge Device V1.5.0-21-amd64
-- OPC UA Connector V1.7.0-15
-- Databus V2.0.0
-- External Databus V2.0.0
-- Data Service V1.6.0
-- IE Flow Creator V1.3.9
-- Energy Manager V1.2.0
-- IE Cloud Connector V1.6.2
-- IIH Core V1.6.0
-- IIH Configurator V1.6.0
+- Industrial Edge Management Virtual V2.0.1-1
+- Industrial Edge Virtual Device V1.16.1-1-a
+- OPC UA Connector V2.0.1-0
+- Registry Service V1.9.0-0
+- Common Import Converter V2.1.0-2
+- Databus V2.3.2-5
+- IIH Essentials V1.10.0
+- Flow Creator V1.17.0-2
+- Energy Manager V1.16.1
+- DataXess V1.4.1-3
+- IIH Semantics V1.9.0-0
+- Common Configurator V1.9.0-4
 - Web browser (Mozilla or Chrome)
 
+- Common Connector Configurator V1.9.1-1
+- DataXess Configurator V1.4.2-4
```

Comment:
Fix spacing of list.
Used Components and Configuration North and Southbound spacing is different to Prerequisites
![image](https://github.com/industrial-edge/Shopfloor-to-Cloud-Connectivity/assets/74045802/0accefaa-5d0e-4507-8a86-e111848cd9ce)

Reply:
Done